@navikt/core/react/src/form/combobox/Input/Input.tsx

```diff
@@ -46,28 +46,40 @@ const Input = forwardRef<HTMLInputElement, InputProps>(
 
     const onEnter = useCallback(
       (event: React.KeyboardEvent) => {
+        const isTextInSelectedOptions = (text: string) => {
+          return selectedOptions.find(
+            (item) => item.toLowerCase() === text.toLowerCase()
```

Comment:
Change to toLocaleLowerCase to account for differences in some languages, like Turkish:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/toLocaleLowerCase#description